main.go

```diff
@@ -1,4 +1,4 @@
-// +build go1.14
```

Comment:
Not needed. `go1.14` is set on all versions following v1.14.

Reply:
...or is it used as a min version check? You can use the go.mod version for that.

Reply:
yes its a min version check, its old style removing it @klauspost